content/english/anti-vegan/humans-are-superior.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Humans Are Superior"
+date: 2019-10-12T07:28:15+13:00
+draft: false
+description: "How to respond to the argument: We are entitled to use animals as a resource because humans are superior to animals"
+weight: 5
+---
+# Argument
+
+> ...but, humans are superior...  
+>   
+> We are entitled to use animals as a resource because humans are superior to animals
+
+# Rebuttal
+
+## Quick Response
+
+> Superiority supposes an arbitrary line that identifies one group as deservedly superior to another. Common examples are that men are superior to women (sexism) and whites are superior to all other ethnicities (racism).  
+>   
```

Comment:
Can we separate the arguments with the word `or` and without the `> ` prefix on the line? Otherwise it doesn't render as separate quote blocks in the app.

Reply:
'Discorded... not sure on any options here / what works what doesn't